src/components/ItaliaTheme/Blocks/Accordion/Edit.jsx

```diff
@@ -45,6 +55,42 @@ class Edit extends SubblocksEdit {
           <Container className="px-md-4">
             <Card className="card-bg rounded" noWrapper={false} space tag="div">
               <CardBody tag="div">
+                <TextEditorWidget
+                  data={this.props.data}
+                  fieldName="title"
+                  selected={this.state.selectedField === 'title'}
+                  block={this.props.block}
+                  onChangeBlock={(data) => {
+                    this.props.onChangeBlock(this.props.block, {
+                      ...data,
+                    });
+                  }}
+                  placeholder={this.props.intl.formatMessage(messages.title)}
+                  showToolbar={false}
+                  onSelectBlock={() => {}}
+                  onAddBlock={() => {
+                    this.setState({ selectedField: 'description' });
+                  }}
+                />
+                <div className="description">
```

Comment:
a cosa serve questo div? in teoria lo mette gia il widget il div con il nome del campo